docs/jitserver.md

```diff
@@ -0,0 +1,88 @@
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# JITServer technology (technical preview)
+
+![Start of content that applies to Java 8 and later](cr/java8.png)
+![Start of content that applies to Java 11 and later](cr/java11.png)  
+**Linux&reg; on x86-64 only**
+
+JITServer technology decouples the JIT compiler from the VM and lets the JIT compiler run remotely in its own process. This prevents your Java&trade; application suffering possible negative effects due to CPU and memory consumption caused by JIT compilation.
+
+This technology can potentially improve quality of service, robustness, and even performance of Java applications. We recommend trying this technology if the following criteria are met:
+
+- Your Java application is required to compile many methods using JIT in a relatively short time.
+- The application is running in an enviroment with limited CPU or memory, which can worsen interference from the JIT compiler.
+- The network latency between JITServer and client VM is relatively low.
+
+For more details about JITServer technology, its pros and cons, and when best to use it, see the blog [Free your JVM from the JIT with JITServer Technology](https://blog.openj9.org/2020/01/09/free-your-jvm-from-the-jit-with-jitserver-technology/).
+
+## Using JITServer technology
+
+OpenJ9 builds available from [AdoptOpenJDK](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=openj9) have JITServer technology included, though you must explicitly invoke it.
+
+To use JITServer technology, use one of the following options:
+
+|  Setting            | Effect                                                             |
+|---------------------|--------------------------------------------------------------------|
+| none                | Launches OpenJ9 as a regular VM with embedded JIT compilation      |
+| `-XX:+UseJITServer` | Launches OpenJ9 in client mode (**Note 1**)                        |
+| `jitserver`         | Launches OpenJ9 in JITServer mode (**Note 2**)                     |
+
+**Note 1:** With this command-line option, the VM sends compilation requests to an available JITServer. The client operates as a regular VM if a server is not available.  
+**Note 2:** Use the `jitserver` command to start a VM that listens for incoming compilation requests.
```

Comment:
This shouldn't say "to start a VM". It starts the JIT server.

Reply:
Implementation wise we are actually creating and starting a JVM, but it can be confusing. Maybe just say 'it starts a JITServer process that listens for incoming compilation requests'